### PR TITLE
French json file for latest campaign Scarlet Keys + translations + corrections

### PR DESCRIPTION
**Pull request by fragarch**
All Survivor cards are now done

### CODE REVIEW

File: translations/fr/pack/tsk/tskp.json [hunk 1]
```diff
@@ -332,8 +332,8 @@
         "skill_intellect": 5,
         "skill_combat": 2,
         "skill_willpower": 2,
-        "subname": "The Photographer",
-        "text": "You begin the game with Darrell's Kodak in play.\n[fast] During a skill test at your location, spend 1 evidence from an asset you control: Reduce the difficulty of this test by 2. (Limit once per test.)\n[elder_sign] effect: +1. Place 1 evidence on an asset you control.",
+        "subname": "Le Photographe",
+        "text": "Vous commencez la partie avec le Kodak de Darrell en jeu.\n[fast] Lors d'un test de compétence dans votre lieu, dépensez 1 preuve d'un soutien que vous controlez : diminuez de 2 la difficulté de ce test. (Limite d'une fois par test.)\n[elder_sign] effect: +1. Placez 1 preuve sur un soutien que vous contrôlez.",
```
Comment (cfergeau) [inline translations/fr/pack/tsk/tskp.json:336]: > `d'un soutien que vous controlez`
> `Placez 1 preuve sur un soutien que vous contrôlez.`

Different spellings for `contrôlez` ?

Comment (fragarch) [reply translations/fr/pack/tsk/tskp.json:336]: je vais corriger, bien vu

Comment (fragarch) [reply translations/fr/pack/tsk/tskp.json]: I corrected and added two signature cards of Darrell

**Comment by fragarch** [comment]
> Only one card done (Darrell Simmons) as I'm new here and test how it works ;)

Now around 10 cards done.

File: translations/fr/pack/tsk/tskp.json [hunk 2]
```diff
@@ -1598,16 +1598,16 @@
...
-        "name": "Hidden Pocket",
+        "name": "Poche Dissimulée",
         "pack_code": "tskp",
         "position": 65,
         "quantity": 2,
         "skill_agility": 1,
         "skill_intellect": 1,
-        "text": "Fast. Play only during your turn.\nAttach to a [[Clothing]] or [[Armor]] asset you control.\nAttached asset gains: \"You have 1 additional hand or accessory slot (your choice), which can only be used to hold an [[Illicit]] asset.\"",
-        "traits": "Upgrade. Illicit.",
+        "text": "Rapide. À jouer uniquement lors de votre tour.\nAttachez cette carte à un soutien [[Vêtement]] ou [[Armure]] que vous you contrôlez.\nLe soutien attaché gagne : \"Vous avez 1 emplacement de main ou d'accessoire supplémentaire (vous choisissez) qui ne peut être utilisé que pour contenir un soutien [[Illicite]].\"",
```
Comment (cfergeau) [inline translations/fr/pack/tsk/tskp.json:1609]: `que vous you contrôlez`, le `you` est de trop :)

**Comment by fragarch** [comment]
@Kamalisk Hello, is there anything missing to approve those commits ? I plan on finishing the last cards of latest expansion in the coming days (probably Monday) and will then issue a new pull request. Thanks !!

**Comment by fragarch** [comment]
@Kamalisk All cards from Scarlet Keys are done in French. would be nice if you could merge those texts. Thanks

**Pull request merged**